app/src/main/java/com/stevesoltys/seedvault/restore/RestoreViewModel.kt

```diff
@@ -201,18 +204,11 @@ internal class RestoreViewModel(
         }
 
         // we need to retrieve the restore sets before starting the restore
-        // otherwise restoreAll() won't work as they need the restore sets cached internally
-        val observer = RestoreObserver { observer ->
```

Comment:
Nice, so we don't need to retrieve the restore sets before anymore?

Reply:
Am I missing something, or are you saying this because the comment appears to have been removed? It wasn't removed, it was modified to say `restorePackages()` instead of `restoreAll()`, since this changes what we are calling.

Reply:
Ah it is down there. So we still need to do that, did you try?